build.sbt

```diff
@@ -1,8 +1,8 @@
 import sbt.Keys._
 
-ThisBuild / scalaVersion := "2.12.13"
+ThisBuild / scalaVersion := Settings.scala212
 ThisBuild / organization := "org.locationtech.geotrellis"
-ThisBuild / crossScalaVersions := List("2.12.13", "2.11.12")
+ThisBuild / crossScalaVersions := List(Settings.scala213, Settings.scala212, Settings.scala211)
```

Comment:
I know that it is premature, but I'd drop 2.11 and move towards Spark 3 (see https://github.com/locationtech/geotrellis/pull/3294); we have a greenlight here since GeoMesa also crosscompiled against scala 2.12 now and is able to work with Spark 3.0.x

Reply:
I think that's the right thing to do when the spark upgrade happens.